src/gui/pattern.cpp

```diff
@@ -934,6 +934,21 @@ void FurnaceGUI::drawPattern() {
           ImGui::TableNextColumn();
         }
       }
+      // release selection if mouse released
```

Comment:
This should be put in FurnaceGUI::loop() instead, because if the pattern window is closed, collapsed or too small, this will not execute and therefore selection status will never change.

Reply:
Okay, have pushed a commit that relocates it.